ir_private.h

```diff
@@ -133,9 +143,9 @@ IR_ALWAYS_INLINE uint32_t ir_ntzl(uint64_t num)
 	unsigned long index;
 
 #if defined(_WIN64)
-	if (!BitScanForward64(&index, num)) {
+	if (!_BitScanForward64(&index, num)) {
 #else
-	if (!BitScanForward(&index, num)) {
+	if (!_BitScanForward(&index, num)) {
 #endif
```

Comment:
I'm not sure if ``BitScanForward()`` works for ``uint64_t``. I know, this code was here before this patch.

Reply:
Perhaps it's not that bad if the values are expected to be in the range. But depends on what is passed in here, maybe it's indeed better to have a different signature. I'll verify more on this.

Thanks

Reply:
I made some ifdef at another place also, for `ir_nlz`, as 64-bit version doesn't exist in x86 build. Perhaps it'd be cleaner, for 64-bit input, to just have the plain C implementation. Maybe it's a better way to do that.

Thanks

Reply:
I think, use clean C version for uint64_t on 32-bit Windows is  acceptable. It shouldn't be used at all.